sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -231,14 +231,15 @@ public virtual RecognizeReceiptsOperation StartRecognizeReceiptsFromUri(Uri rece
         /// <returns>A <see cref="RecognizeCustomFormsOperation"/> to wait on this long-running operation.  Its <see cref="RecognizeCustomFormsOperation"/>.Value upon successful
         /// completion will contain extracted pages from the input document.</returns>
         [ForwardsClientCalls]
-        public virtual RecognizeCustomFormsOperation StartRecognizeCustomForms(string modelId, Stream formFileStream, /* ContentType contentType, */ RecognizeOptions recognizeOptions = default, CancellationToken cancellationToken = default)
+        public virtual RecognizeCustomFormsOperation StartRecognizeCustomForms(string modelId, Stream formFileStream, RecognizeOptions recognizeOptions = default, CancellationToken cancellationToken = default)
         {
-            throw new NotImplementedException();
+            recognizeOptions ??= new RecognizeOptions();
+
+            // TODO: automate content-type detection
+            // https://github.com/Azure/azure-sdk-for-net/issues/10329
 
-            //// TODO: automate content-type detection
-            //// https://github.com/Azure/azure-sdk-for-net/issues/10329
-            //ResponseWithHeaders<AnalyzeWithCustomModelHeaders> response = ServiceClient.AnalyzeWithCustomModel(new Guid(modelId), includeTextDetails: includeTextElements, formFileStream, contentType, cancellationToken);
-            //return new RecognizeFormsOperation(ServiceClient, modelId, response.Headers.OperationLocation);
+            ResponseWithHeaders<ServiceAnalyzeWithCustomModelHeaders> response = ServiceClient.RestClient.AnalyzeWithCustomModel(new Guid(modelId), ContentType.Pdf, formFileStream, includeTextDetails: recognizeOptions.IncludeTextContent, cancellationToken);
```

Comment:
Not related to this PR but looking at `includeTextDetails: recognizeOptions.IncludeTextContent,` made me wonder, is there any reason why our name is different than the param?

Reply:
We've renamed this a bunch of times (rawExtractedFields, textElements, etc).  The idea with this iteration is now we have FormContent and TextContent in various places in the API, so it is using the same name in the parameter as what you'll get populated in the various data-holding types.